src/MOI/MOI_wrapper.jl

```diff
@@ -3695,6 +3695,85 @@ function MOI.supports(
     return true
 end
 
+struct IntegerAttribute <: MOI.AbstractOptimizerAttribute
```

Comment:
Perhaps 
```Julia
struct Attribute{T} <: MOI.AbstractOptimizerAttribute
```

Then you can have `Attribute{Integer}`, `Attribute{String}`, and `Attribute{Real}`. Same for `struct Control{T}`.

Reply:
Good point. I am considering removing these in favor of `get_control_or_attribute` from latest commit.
What do you think?